awx/main/migrations/_create_collection_system_jt.py

```diff
@@ -0,0 +1,42 @@
+import random
+import logging
+
+from django.db import migrations, models
+from django.utils.timezone import now, timedelta
+
+logger = logging.getLogger('awx.main.migrations')
+
+__all__ = ['create_system_job_templates']
+
+def create_system_job_templates(apps, schema_editor):
+    '''
+    Create default system job templates if not present. Create default schedules
+    only if new system job templates were created (i.e. new database).
+    '''
+
+    SystemJobTemplate = apps.get_model('main', 'SystemJobTemplate')
+    Schedule = apps.get_model('main', 'Schedule')
+    ContentType = apps.get_model('contenttypes', 'ContentType')
+    sjt_ct = ContentType.objects.get_for_model(SystemJobTemplate)
+    now_dt = now()
```

Comment:
I'm fine w/ the `+-30 minute` before to stagger these, but we should probably aim _somewhere_ around 9pm-3am EST as a default.

I don't think having these run by default during US/European business hours is the best choice.

Reply:
:+1: I have changed this to set the start time for the schedule as midnight the night of the job creation.